### ISSUE TITLE
Transfers in web interface

### ISSUE THREAD

**Issue by CaptainFrosty** [mtotschnig/MyExpenses#912]
Name says it all. Just would like to do transfers in the web interface.

**Comment by ItsIgnacioPortal** [comment]
Agreed. We can already do expenses and incomes, but we can't do transfers between accounts. Please add this feature!
